init.lua

```diff
@@ -117,10 +135,20 @@ function SkyRocket:isMoving()
   return self.dragType == dragTypes.move
 end
 
+function SkyRocket:isFancy()
+  return self.dragType == dragTypes.fancy
+end
+
 function SkyRocket:handleDrag()
   return function(event)
     if not self.dragging then return nil end
 
+    if not self.showcanvas then
```

Comment:
Is the intention here "if we're not showing the canvas, show it"?

If so, name the variable `showingCanvas` – it makes more linguistic sense.

(`showCanvas` implies that "in the future, maybe we should show the canvas if it's `true`" which I think is _not_ your intention…)